### ISSUE TITLE
Out of memory error when click next/prev

### ISSUE THREAD

**Issue by safe-bug** [hannesa2/panoramagl#97]
Hello

We are using the library to show tiled images, we fetch each tile then create a bitmap for each, and then merge them to create the whole bitmap. But when we click next/prev buttons a few times we get out of memory error. We recycle the previous bitmaps on every click on the buttons, but on the profiler, it shows that they're still existing in the ram.

Can you help me to fix this issue?

Thanks

**Comment by hannesa2** [comment]
It has the touch of #79
Please make a pull request _here_ where I can easy reproduce the issue. 
I mean not: sometimes it has this error. 
I mean: let something iterate and the I see ... what ? 

Exactly, show me your logcat entry of this error. Even #79 never show me the _exact_ error

**Comment by mupm1** [comment]
I made a loop that changes the image every second.
The memory fills up and the application closes. Is there a solution to this?

[my code is here](https://github.com/mupm1/panoramagl/tree/NextStep)

[Profiler image](https://i.postimg.cc/MKSfMxmT/1.png)

**Comment by hannesa2** [comment]
Thank you for the sample, I squashed all 3 commit s to see an overall of your changes. Because of missing permission to push to your branch I create this pull request #107 with this changes
* squash all your commits
* remove old build leftovers in Gradle files

<img width="1370" alt="image" src="https://user-images.githubusercontent.com/3314607/136747785-f50e42d0-2b6a-4a63-8160-805ac730cea1.png">
Sorry I didn't see the issue and here is the complete movie  

https://user-images.githubusercontent.com/3314607/136747924-a86c0986-cd0a-4ef3-8b38-66ea6c89d848.mov

### A
Please test my pull request. #107

### B
Compare to my screencast, what did you different ?

**Comment by hannesa2** [comment]
Hmm, I use a api30 device (see screenshot) and you ?

**Comment by mupm1** [comment]
I use android 11. Samsung A70

**Comment by mupm1** [comment]
Can you test on a real device?

**Comment by hannesa2** [comment]
Indeed, on a Pixel 4 running Android 11 I observe it. But it's not dying 
<img width="775" alt="image" src="https://user-images.githubusercontent.com/3314607/136895215-8f4e41f1-9ea1-44f7-b624-b9c6bcbe2f6b.png">

**Comment by hannesa2** [comment]
Now it becomes better, but not good
<img width="770" alt="image" src="https://user-images.githubusercontent.com/3314607/136900592-4c4ba33b-50c1-46fb-9acf-e11eab1f7e4c.png">

**Comment by hannesa2** [comment]
Now, it comes with this memory consumption on a Pixel 4
![image](https://user-images.githubusercontent.com/3314607/137079787-a0acd37f-102b-4135-9906-b7c1ac802300.png)

@mupm1 @safe-bug please test with your device this branch #107

**Comment by mupm1** [comment]
it works great. thanks
![Capture](https://user-images.githubusercontent.com/7701890/137089072-4855f497-3873-412e-b535-0115009931c2.PNG)
